node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/configserver/noderepository/RealNodeRepository.java

```diff
@@ -83,17 +85,33 @@ public RealNodeRepository(ConfigServerApi configServerApi) {
     }
 
     @Override
-    public List<AclSpec> getNodesAcl(String hostName) {
+    public Map<String, Acl> getAcl(String hostName, Set<String> containerNames) {
```

Comment:
Why do we still have the second argument? Its not needed, and will this even work at all? This tries to match container names against `trustedBy`s, which is hostnames? So this will always return empty map?

Reply:
Yeah, the original idea was to ask for individual acls as that made the logic on the caller side mush cleaner... but as that is not feasible I'll revert and add the hostname to the Acl class and make that class a little bit smarter with respect to net family presence (help solves the if address not present thing).

Reply:
No, you can get the exact same return without sending the list of containers, see the long code snippet i wrote in this PR on April 9th.

Reply:
Well the point was to explicitly tell which ACL we wanted and not all... but changed this in later commit.